classes/helper/HelperList.php

```diff
@@ -269,6 +269,8 @@ public function displayListContent()
                 }
             }
 
+            $this->_list[$index]['onclick'] = in_array('view', $this->actions) ? $this->getViewLink($this->token, $id) : $this->getEditLink($this->token, $id);
```

Comment:
Shouldn't this be `link` instead of `onclick`?

Reply:
This is linked to the modif I made in the template where the "onclick" url was generated.., I wanted to keep in mind the use of this variable

https://github.com/PrestaShop/PrestaShop/pull/14721/files#diff-a3eeb66715b0f2f29851fb427bf72e14R52

But I can rename it with `link` and adapt the template accordingly if you prefer?

Reply:
I think it would be best since it's semantically wrong. Its value is not something that you'd put in onclick directly, it's an URL.